py2many/rewriters.py

```diff
@@ -172,13 +172,17 @@ def visit_If(self, node):
                 rename(node.scopes[-2], "main", "main_func")
             # ast.parse produces a Module object that needs to be destructured
             if self.main_signature_arg_names == {"argc", "argv"}:
-                ret = ast.parse(
-                    "def main(argc: int, argv: List[str]) -> int: True"
-                ).body[0]
+                ret = cast(
+                    ast.FunctionDef,
+                    ast.parse("def main(argc: int, argv: List[str]) -> int: True").body[
```

Comment:
IMO, anywhere this ugly `[ \n   0 \n    ]` appears, we need a helper.

Reply:
Yeah - black makes it worse.